src/Compatibility/Android.FormsViewGroup/src/Compatibility.Android.FormsViewGroup.csproj

```diff
@@ -1,16 +1,14 @@
-<Project Sdk="MSBuild.Sdk.Extras">
```

Comment:
This may have been the source of some grief of this one not compiling nicely for us since it used sdk extras...